lib/Room.php

```diff
@@ -279,6 +284,10 @@ public function setListable(int $newState): void {
 		$this->listable = $newState;
 	}
 
+	public function getTimeToLive(): int {
+		return $this->timeToLive;
+	}
```

Comment:
Needs a `setTimeToLive` method as well (see new way of doing things from https://github.com/nextcloud/spreed/issues/6235